spec/built-in-modules/color.md

```diff
@@ -711,3 +729,5 @@ whiteness($color)
   > RGB channels and convert back when `whiteness()` or `blackness()` is called,
   > where another may keep around their original HWB values and return those
   > as-is.
+
+[special variable string]: ../spec.md#special-variable-string
```

Comment:
We generally put reference links after the first reference so they're easier to clean up later on. Also, I think this term is defined in `functions.md`.